changelog.md

```diff
@@ -1,8 +1,11 @@
 ## Change from 1.6.352 to 1.6.xxx
 
-## Limit maximum number of running deployments
+### Limit maximum number of running deployments
 New command line flag `--max_running_deployments` was added to limit the max number of concurrently running deployments. The default value is set to 100. Should the user try to submit more updates than set by this flag a HTTP 403 Error is returned with an explanatory error message. We introduced this flag because having lots of running deployments can lead to a significant performance decrease in the failover scenario during marathon initialization phase. 
 
+### Zookeeper garbage cleaning interval
+New command line flag `--gc_actor_cleaning_interval` was added to set cleaning interval of GcActor in seconds. The default value is set to 30 seconds.
```

Comment:
Feels like we expose an implementation detail by having `gc_actor` as a part of the whole parameter name. What do you think about something like `storage_gc_interval`?

Reply:
Oh, boy, there is another parameter named like this. I guess, we should keep it as is for the sake of consistency.

Reply:
> yeah, agree, I would drop the word "actor" from this

that could give somebody an impression that we trigger JVM's GC which is not the case here. `gc_actor_` is not optimal but `gc_` is arguably even worth.

> Oh, boy, there is another parameter named like this. I guess we should keep it as is for the sake of consistency.

The other parameter you're talking about is `gc_actor_scan_batch_size` and it was also introduced recently and *hasn't been released* yet. So if want to come up with the right naming schema - now is the time /cc @ichernetsky

Reply:
Great then. What about `storage_compaction_interval` and `storage_compaction_scan_batch_size`?